template/utils/send_sample_request.js

```diff
@@ -128,7 +134,8 @@ define([
             data       : param,
             type       : type.toUpperCase(),
             success    : displaySuccess,
-            error      : displayError
+            error      : displayError,
+            ...bodyFormDataOtherConfig
```

Comment:
This is an ES6 feature, it shouldn't appear here AFAIK as the rest of the codebase is not ES6 nor transpiled by babel before the output.